source/database-connections/save-mongodb-connection.txt

```diff
@@ -0,0 +1,76 @@
+
+.. _rm-save-mongodb-connection:
+
+============================================
+Save a MongoDB Connection
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To start a migration job, Relational Migrator must connect to your MongoDB
+database using an account with :authrole:`readWrite` access. You can save a new
+MongoDB Connection:
+
+- When creating or modifying a :ref:`data migration job <rm-create-jobs>`
+- From the Database Connections bar in a project
+- From the Relational Migrator :guilabel:`Connections` page
+
+Before you Begin
+----------------
+
+- To save a connection to a MongoDB Atlas cluster, you must be :ref:`logged in <rm-atlas-log-in>` with Atlas.
+
+- For both Atlas and on-premises deployments, we recommend creating a separate
+  :ref:`MongoDB user <rm-mongodb-service-user>` for Relational Migrator.
+
+Steps
+-----
+
+To save a new connection from the :guilabel:`Connections` page:
+
+.. procedure::
+
+   .. step:: Add a new connection
+      
+      In the list of :guilabel:`MongoDB` connections, click :guilabel:`+ Add connection`.
+
+   .. step:: Enter the MongoDB connection string
+
+      a. In the :guilabel:`MongoDB connection string (URI)` field, input
+         your :manual:`MongoDB URI <mongodb-uri>`.
+
+      #. If it isn't included in the connection string, enter the
+         :guilabel:`Database` to connect to.
+      
+      #. If they aren't included in the connection string, enter the 
+         :guilabel:`Username` and :guilabel:`Password` of your
+         :ref:`Relational Migrator MongoDB user <rm-mongodb-service-user>`.
+         
+         Checking :guilabel:`Save password` saves the password securely
```

Comment:
Thoughts here?

Reply:
Forgot to write up my thoughts when I was going through the first time -- I have it in the step description since it's an optional choice that modifies the result of the main step. With the table format of fields for relational DBs, as a comparison, the checkbox isn't its own line for the same reason. Does that make sense?